lib/src/builder.dart

```diff
@@ -78,10 +78,13 @@ class GeneratorBuilder extends Builder {
       genPartContent = formatter.format(genPartContent);
     } catch (e, stack) {
       buildStep.logger.severe(
-          """Error formatting the generated source code.
-This may indicate an issue in the generated code or in the formatter.
-Please check the generated code and file an issue on source_gen
-if approppriate.""",
+          ''
+          'Error formatting generated source code for ${library.identifier} '
+          'which was output to ${_generatedFile(buildStep.inputId).path}.\n'
+          'This may indicate an issue in the generated code or in the formatter.'
+          '\n'
```

Comment:
We've been moving to this format in a lot of places. `'''` works best for things like top-level variables where the string doesn't have any long lines

- let's us break up the String arbitrarily and and separate from how it's broken in the output to fit 80 chars
- Doesn't introduce a weird jump to the left of the screen from an indented block when the output *shouldn't* be indented